.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Question
+about: Ask about anything NewPipe-related
+labels: question
+assignees: ''
+
+---
+
+<!-- IF YOU DON'T FILL IN THE TEMPLATE PROPERLY, YOUR ISSUE IS LIABLE TO BE CLOSED. If you feel tired/lazy right now, open your issue some other time. We'll wait. -->
+
+<!-- The comments between these brackets won't show up in the submitted issue (as you can see in the Preview). -->
+
+### Checklist
+<!-- This checklist is COMPULSORY. The first box has been checked for you to show you how it is done. -->
+
+- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. <!-- Seriously, check. O_O (If there's already an issue but you'd like to see if something changed, just make a comment on the issue instead of opening a new one.) -->
+- [ ] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
+- [ ] This issue contains only one question. I will open one issue for every question I want to ask.
```

Comment:
I'm a bit hesitant about having this item here. What do you think?

Reply:
I guess that multiple questions per issue are not a problem.